tests/testing-framework/src/runtimes/in_process_spin.rs

```diff
@@ -104,7 +104,7 @@ async fn initialize_trigger(
     .await?;
 
     let app = spin_app::App::new("my-app", locked_app);
-    let trigger = HttpTrigger::new(&app, "127.0.0.1:80".parse().unwrap(), None)?;
+    let trigger = HttpTrigger::new(&app, "127.0.0.1:80".parse().unwrap(), None, true)?;
```

Comment:
Do we want to end up on a random port here?  I am not sure if the testing infra will be happy with that.  Although I guess it should never happen.  But it might be better for it to fail here if it did.